python/lsst/ts/mtaos/mtaos.py

```diff
@@ -203,6 +225,10 @@ def __init__(
         self.m2_min_forces_to_apply = 1e-3
 
         self.ocps = salobj.Remote(self.domain, "OCPS", 101)
+        self.mtcs = salobj.Remote(self.domain, "MTCS", 101)
```

Comment:
This is not right.. you are probably trying to instantiate the `MTCS` class from `ts_observatory_control` no?

But also, note we already have remotes for M1M3, M2, and both hexapods. So, either we will have to refactor the MTAOS to use the MTCS or use the remotes directly.  See above the `self.remotes` attribute.

Reply:
Removed mtcs since it wasn't really needed.